app/commands.py

```diff
@@ -480,41 +466,16 @@ async def _with(ctx: Context) -> str | None:
 
     mode_vn = ctx.player.last_np["mode_vn"]
 
-    command_args = parse__with__command_args(mode_vn, ctx.args)
-    if isinstance(command_args, ParsingError):
-        return str(command_args)
-
-    msg_fields = []
-
-    score_args = ScoreParams(mode=mode_vn)
-
-    mods = command_args["mods"]
-    if mods is not None:
-        score_args.mods = mods
-        msg_fields.append(f"{mods!r}")
-
-    nmiss = command_args["nmiss"]
-    if nmiss:
-        score_args.nmiss = nmiss
-        msg_fields.append(f"{nmiss}m")
-
-    combo = command_args["combo"]
-    if combo is not None:
-        score_args.combo = combo
-        msg_fields.append(f"{combo}x")
-
-    acc = command_args["acc"]
-    if acc is not None:
-        score_args.acc = acc
-        msg_fields.append(f"{acc:.2f}%")
+    score_args = parse__with__args(mode_vn, ctx.args)
+    if isinstance(score_args, ParsingError):
+        return str(score_args)
 
     result = app.usecases.performance.calculate_performances(
         osu_file_path=str(BEATMAPS_PATH / f"{bmap.id}.osu"),
         scores=[score_args],  # calculate one score
     )
 
-    return "{msg}: {pp:.2f}pp ({stars:.2f}*)".format(
-        msg=" ".join(msg_fields),
+    return "{pp:.2f}pp ({stars:.2f}*)".format(
```

Comment:
I think we should keep the `msg` here is nice in the sense that from in-game, it's easy to tell whether or not the server correctly parsed the params you send

It'll be quite nice now that the parsing is split into a function -- we can have a `format__with__args(score_args) -> str` and test that similarly